zowe/zowe-cli
# Attach authentication header to proxy agent

**What It Does**


Change the current way the authorization header is attached for the proxy request to instead be attached in the agent itself to avoid 403 error.

**How to Test**


In order to test this, use a proxy available that requires authentication and follow these setup steps:

1. In the proxy settings, define the url that requires authentication
2. In the proxy authorization setting click `Edit in settings.json` and replace the `null` property with the authentication type followed by space and the credentials in base64. Example: `Basic ==MyBase64UserPassword`. Note the credentials should use this pattern before encoding `user:password`

**Review Checklist**
I certify that I have:


**Additional Comments**
